packages/core/src/components/Image/index.tsx

```diff
@@ -38,7 +40,7 @@ const styleSheet: StyleSheet = ({ ui }) => ({
   fadeIn: {
     animationName: {
       from: {
-        opacity: 0,
```

Comment:
small bump allows us to see a little bit of the image in happo, which is helpful for the alignment and sizing. even with a large happo delay, this fadeIn animation doesn't complete.